docs/GHA.md

```diff
@@ -63,4 +63,4 @@ The Buildkite Migration tool's currently supported (✅), partially supported (
 
 | Key | Supported? | Notes |
 | --- | ---------- | ----- |
-| `run-name` | ❌ | Build messages in Buildkite are set as the `BUILDKITE_MESSAGE` environment variable (commit message from source control). Build messages are settable in manual build creation, and via both [REST](https://buildkite.com/docs/apis/rest-api/builds#create-a-build) and [GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/buildcreate) APIs. |
\ No newline at end of file
+| `run-name` | ❌ | Build messages in Buildkite are set as the `BUILDKITE_MESSAGE` environment variable (commit message from source control). Build messages are settable in manual build creation, and via both [REST](https://buildkite.com/docs/apis/rest-api/builds#create-a-build) and [GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/buildcreate) APIs. |
```

Comment:
I think this is just my local editor settings dropping a newline at the end of the file - feel free to ignore this particular change if we don't want it.

Reply:
All good - docs crossover recently too, thanks @pbuckley 💚